tools/run_tests/run_interop_tests.py

```diff
@@ -816,6 +836,11 @@ def aggregate_http2_results(stdout):
                   action='store_const',
                   const=True,
                   help='Enable HTTP/2 server edge case testing. (Good client, bad server)')
+argp.add_argument('--http2_edge_server_interop',
```

Comment:
these work with different sets of client languages - but I think they can probably still be combined, WIP on that